### DISCUSSION TITLE
Work in progress website to learn Closeread

### DISCUSSION THREAD

**Discussion by gastonstat** [qmd-lab/closeread#152] [Show and tell]
Hi all. I just wanted to share a work in progress 🚧 website for learning **Closeread** (in addition to the official documentation's website [closeread.dev](https://closeread.dev/)) that I plan to use with my students to teach them how to create scrollytelling stories with Closeread

[gastonsanchez.com/learn-closeread](https://www.gastonsanchez.com/learn-closeread)

<a href="https://www.gastonsanchez.com/learn-closeread"><img width="700" alt="learn-closeread" src="https://github.com/user-attachments/assets/4468ca36-e6e7-4f0e-b222-4d4d1a19e61c"></a>

**Comment by jimjam-slam** [comment]
This is so rad, @gastonstat! Please consider entering it in the [Closeread Prize](https://posit.co/blog/closeread-prize-announcement/)! 🥳

**Comment by andrewpbray** [comment]
This is terrific @gastonstat ! Those diagrams that you've put together are excellent!
